compiler/luci/pass/src/QuantizationUtils.cpp

```diff
@@ -78,15 +85,15 @@ void compute_asym_scale_zp(float min, float max, float &scaling_factor, int64_t
   }
   else
     zero_point_double = qmin_double - rmin / scale;
-  if (zero_point_double <= qmin_double)
+  if (min >= 0)
   {
     assert(min >= 0 && max >= 0);
     nudged_zero_point = kMinScale;
     scale = max / (qmax_double - qmin_double);
```

Comment:
Is this line necessary?

if min >= 0, it seems that the scale is the same as the value set in line 75.

`double scale = (rmax - rmin) / (qmax_double - qmin_double);`

Reply:
I think line 100 can be removed as well.